plugins/modules/pnp_intent.py

```diff
@@ -421,6 +437,17 @@ def get_claim_params(self):
             ]
         }
 
```

Comment:
self.validated_config[0]["template_params"]  is used in 4 places, can we simplify the code?

```
template_params = self.validated_config[0].get("template_params", {})

if configinfo["configId"] and template_params and isinstance(template_params, dict):
    configinfo["configParameters"] = []
    for key, value in template_params.items():
        config_dict = {'key': key, 'value': value}
        configinfo["configParameters"].append(config_dict)
```